src/bundle/Resources/public/js/scripts/sidebar/btn/content.edit.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    const editButton = document.querySelector('.ez-btn--edit[data-actions="edit"]');
+
+    editButton.addEventListener('click', () => {
+        const languageButtons = document.querySelectorAll('.ez-extra-actions--edit[data-actions="edit"] .form-check [type="radio"]');
+
+        if (languageButtons.length !== 1) {
```

Comment:
When there is a possibility that there are more than 1 buttons? Or maybe, you're checking if any button exists?

Reply:
When there is more than one language added for content, probably languagebuttons for it is not the best name then, gonna change it.